auth/provider/types_test.go

```diff
@@ -0,0 +1,23 @@
+package provider
+
+import "testing"
+
+func TestSupportedProviders_IsSupported_whenProviderIsSupported(t *testing.T) {
+	if !IsSupported("github") {
+		t.Errorf("Expected: \"%v\". Got: \"%v\"", true, false)
```

Comment:
The terminology in got is want, got. Please see the code in openfaas/faas openfaas/faas-cli for examples.

Reply:
Changed